modules/i18n/src/translations/en-US/portals/user.ts

```diff
@@ -88,6 +88,14 @@ export const user: userNS = {
                 part1:"Enter the",
                 part2: "in the format YYYY-MM-DD"
             }
+        },
+        userActionZoneGroup: {
+            header: "User Action",
+            impersonateUserZone: {
+                actionTitle: "Impersonate User",
+                header: "Impersonate User",
+                subheader: "Once Started impersonating the user, you will no longer be able to login with your identity until you terminated the session."
```

Comment:
Let's use the third person narrative, if possible

Reply:
This is the narrative used for other danger zone items as well.

Reply:
Fixed with 82d4ff507d55ec7ed6c0b57080b6e89963594857